<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>681</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Выбор дежурных</string>
     </attribute>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>571</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
	border-radius: 10px;
	background-color: rgb(176, 224, 230);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Futura LT</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Дата:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ledit_day_month">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>70</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Futura LT</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>501</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Futura LT</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Ближайшее предстоящее дежурство:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>140</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Futura LT</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Дата:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_end">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>70</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Futura LT</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Завершить</string>
       </property>
      </widget>
     </widget>
     <widget class="QTableWidget" name="tblwg_sur_name_n">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>250</y>
        <width>311</width>
        <height>281</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Фамилия</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Имя</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Дежурил _ раз</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>401</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Futura LT</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Удтвердите трёх учеников на дежурство:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>220</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Futura LT</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ученики Вашего класса:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Futura LT</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ученик 1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Futura LT</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ученик 2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Futura LT</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ученик 3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_1st_st">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>250</y>
        <width>151</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Futura LT</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Выбрать</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_3d_st">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>330</y>
        <width>151</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Futura LT</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Выбрать</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_2nd_st">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>290</y>
        <width>151</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Futura LT</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Выбрать</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_3d_st_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>380</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Futura LT</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Подтвердить</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Личная информация</string>
     </attribute>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>-20</y>
        <width>681</width>
        <height>551</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
	border-radius: 10px;
	background-color: rgb(176, 224, 230);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>70</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Futura LT</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Логин:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_register">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>251</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Futura LT</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Личные данные:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>300</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Futura LT</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Пол:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ledit_login">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>70</y>
         <width>221</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="ledit_surname">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>110</y>
         <width>221</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>110</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Futura LT</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Фамилия:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>150</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Futura LT</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Имя:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ledit_name">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>150</y>
         <width>221</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="ledit_class">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>230</y>
         <width>221</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>190</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Futura LT</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Отчество:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>230</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Futura LT</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Класс:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>380</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Futura LT</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Пароль:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ledit_gender">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>300</y>
         <width>221</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_change_gender">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>340</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Futura LT</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Изменить пол</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_change_password">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>380</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Futura LT</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Изменить пароль</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ledit_fathername">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>190</y>
         <width>221</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_change_class">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>260</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Futura LT</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Изменить класс</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Нежелательные дни</string>
     </attribute>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>671</width>
        <height>521</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
	border-radius: 10px;
	background-color: rgb(176, 224, 230);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="lbl_register_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>591</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Futura LT</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Выберите нежелательные для дежурства дни</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rbtn_mon">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>100</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Futura LT</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Понедельник</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rbtn_tues">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>140</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Futura LT</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Вторник</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rbtn_wend">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>180</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Futura LT</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Среда</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rbtn_fri">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>260</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Futura LT</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Пятница</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rbtn_satur">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>300</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Futura LT</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Суббота</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rbtn_thur">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>220</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Futura LT</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Четверг</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Утвержденные дежурства</string>
     </attribute>
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>611</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
	border-radius: 10px;
	background-color: rgb(176, 224, 230);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="lbl_register_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>541</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Futura LT</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Список утвержденных дежурств:</string>
       </property>
      </widget>
     </widget>
     <widget class="QTableWidget" name="tblwg_day_class">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>150</y>
        <width>311</width>
        <height>331</height>
       </rect>
      </property>
      <row>
       <property name="text">
        <string>Дата1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Дата2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Дата n</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Класс</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>714</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
